src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -1337,7 +1339,10 @@ public void itemCheckedOutEventIsPublished() {
 
     final var checkedOutEvent = publishedEvents.findFirst(byEventType(ITEM_CHECKED_OUT.name()));
 
-    assertThat(checkedOutEvent, isValidItemCheckedOutEvent(loan));
+    JsonObject gracePeriod = loanPolicy.getJson().getJsonObject("loansPolicy")
+      .getJsonObject("gracePeriod");
+
+    assertThat(checkedOutEvent, isValidItemCheckedOutEvent(loan, gracePeriod));
```

Comment:
Let's check that grace period is not sent when there is no grace period in the policy, and add separate test for a case when it exists